tests/stdlib/thttpredirect.nim

```diff
@@ -0,0 +1,41 @@
+discard """
```

Comment:
Network on travis/appveyor isn't reliable so this test will fail often. The best way to test httpclient nowadays is to create an HTTP server locally.

Reply:
i made an attempt to include a local http server in my tests.

Reply:
Brilliant, but this shouldn't need threads. HTTP client and server are both async and thus can operate without blocking without threads.

Reply:
But i wanted to run the tests also in non async httpclients

Reply:
but i also wanted to run the tests using non async httpclients

Reply:
hrm, alright.